### PR TITLE
(feat): Adding minimal permission in openebs-target-pod-failure chart

### LINKED ISSUE #1142: (refactor): Add minimal permission & SvcAcc in openebs-target-pod-failure exp
**What needs to be done**

- Add minimal permission and scope in openebs-target-pod-failure chart - `chaos-chart` repo

- Add the service account details and chaos-engine manifest in  openebs-target-pod-failure docs - `litmus-docs` repo

### PR DESCRIPTION
**Pull request by ispeakc0de**
Signed-off-by: shubhamchaudhary <shubham.chaudhary@mayadata.io>

- Adding minimal permissions in openebs-target-pod-failure experiment

-  Fixes: litmuschaos/litmus#1142

### CODE REVIEW

File: charts/openebs/openebs-target-pod-failure/experiment.yaml [hunk 1]
```diff
@@ -11,31 +11,56 @@ metadata:
   version: 0.1.6
 spec:
   definition:
+    scope: Cluster
     permissions:
       - apiGroups:
           - ""
           - "extensions"
```
Comment (ksatchit) [inline charts/openebs/openebs-target-pod-failure/experiment.yaml:18]: needed?

Comment (ispeakc0de) [reply charts/openebs/openebs-target-pod-failure/experiment.yaml:18]: Yes, to get the resource version of the deployment

**Review by uditgaurav — APPROVED** [review]
lgtm

File: charts/openebs/openebs-target-pod-failure/experiment.yaml (hunk 1 above)
Comment (ksatchit) [reply charts/openebs/openebs-target-pod-failure/experiment.yaml:18]: openebs deployments are still using extensions/v1beta1

**Review by ksatchit — APPROVED** [review]

**Review by chandankumar4 — APPROVED** [review]

**Pull request merged**